__tests__/app.test.js

```diff
@@ -89,6 +89,44 @@ describe('GET:', () => {
 					expect(body.msg).toEqual('Not found');
 				});
 		});
+		describe('/comments', () => {
+			test('status:200 returns a array of objects with the required properties', () => {
```

Comment:
I think this test could be more descriptive, 'returns the comments for the requested article' or something along those lines, it's very generic at the mo! same for the others 😄